### PR TITLE
HLT v2.1 update (81X).

### PR DESCRIPTION
**Pull request by Martin-Grunewald**
HLT v2.1 update (81X).
Based on CMSSW_8_1_X_2016-06-01-1100.

### CODE REVIEW

**Comment by Martin-Grunewald** [comment]
please test

File: RecoTracker/TkTrackingRegions/src/OuterHitPhiPrediction.h [hunk 1]
```diff
@@ -20,7 +20,7 @@ class dso_internal OuterHitPhiPrediction {
       float originRBound) 
     : thePhiAtVertex(phiAtVertex), theCurvature(curvature),
       theOriginRBound (originRBound) {
-       assert(theCurvature.max()>0);
+	//       assert(theCurvature.max()>0);
```
Comment (slava77) [inline RecoTracker/TkTrackingRegions/src/OuterHitPhiPrediction.h]: @VinInn 
please confirm this is OK,
so that it's not masking another problem

Comment (Martin-Grunewald) [inline RecoTracker/TkTrackingRegions/src/OuterHitPhiPrediction.h]: @VinInn @slava77 
See also the comments made very recently at https://github.com/cms-sw/cmssw/pull/13819
which added these "assert"s.

**Comment by Martin-Grunewald** [comment]
+1

**Comment by Martin-Grunewald** [comment]
@Dr15Jones
Also here, could you please sign - the change in Utilities/ReleaseScripts updates the addOn tests.

**Comment by Dr15Jones** [comment]
+1

**Comment by Martin-Grunewald** [comment]
@slava77 @VinInn 
Can you please go ahead (or break 80X as well if this is a real bug...:)

**Comment by cvuosalo** [comment]
+1

For #14811 96eb48ab62ed88c6ab7ea3935d7a993444219ae0

Updating HLT to v2.1.

The code changes are satisfactory. Jenkins tests against baseline CMSSW_8_1_X_2016-06-07-1100 show numerous small differences in HLT DQM quantities, but there appear to be no changes in Reco quantities.

**Comment by VinInn** [comment]
I believe that this is technically speaking a regression ,at least for what concern this PR ,and removing the assert "cures" the regression.
Of course this is curing the symptom not the cause.
I will try to chase the cause, which in my opinion is not related to this PR, and therefore not blocking it...

**Comment by VinInn** [comment]
Please see the discussion in #13819.
Removing the assert is the correct action to reproduce the current behavior at 0T.
That it made any sense is another story....

**Comment by davidlange6** [comment]
+1

**Pull request merged**